composer.json

```diff
@@ -59,7 +59,7 @@
         "twig/extensions": "^1.4",
         "twig/twig": "^1.28",
         "webmozart/assert": "^1.1",
-        "white-october/pagerfanta-bundle": "^1.0",
+        "white-october/pagerfanta-bundle": " ^1.0.8",
```

Comment:
this space should not be there? :wink: 

Caused the `Sylius-Standard` build to fail after updating to latest master version:

https://travis-ci.org/Sylius/Sylius-Standard/jobs/201590967